### DISCUSSION TITLE
Loading error in GitHub repo code page

### DISCUSSION THREAD

**Discussion by askming** [community/community#32708] [Other Feature Feedback, Questions, & Ideas]
Hi,

One of my private repo suddenly couldn't load. I can open the Issues, Actions, etc. pages but it stuck whenever I click on the Code tab and trying to open the load the home page. Keep getting  the screenshot below for the past two days. Can you help?

Thanks!

![image](https://user-images.githubusercontent.com/5671771/189566403-0fd27c5c-dd1d-496f-bd7b-c29bae2f9cc9.png)

**Comment by queenofcorgis** [comment]
Are you still experiencing this? I'd suggest clearing your cookies/cache or using a different browser to see if the problem still persists.

**Reply by askming** [reply]
i just checked, it is resolved now. I tried different browser/computer before, it was the same. 

Thanks for your follow up!

**Reply by queenofcorgis** [reply]
Glad to hear that - thanks for letting me know 🚀
